dt/dt_octavia/DFG-network-octavia-18.0_director-rhel-virthost-3_ocp_workers_2_compute_3_networker-ipv4-geneve-composable.md

```diff
@@ -0,0 +1,53 @@
+# Deployed Topology DFG-network-octavia-18.0_director-rhel-virthost-3_ocp_workers_2_compute_3_networker-ipv4-geneve-composable
```

Comment:
The requirement for testing composable architecture (the one that separates the networker role from the controllers) have existed for a few major versions (I can see that in our d/s docs for OSP 13 we had that as well).

I am not sure what arrived before, the separation of networker role from the controller, or maybe we started with the networker role running separately as a node and then we were able to merge it into the controller, but all networking squads are keeping at least one composable job for a few major versions. I guess some customers like having separation of concerns between the controller and the networker nodes.

I don't know what are going to be the testing requirements regarding NG, but for now I prefer to keep having a composable job definition, just in case.

There are 3 networker nodes on this deployment from the same reason we have 3 controllers, high availability.